Unix/tools/auth_tests_setup.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+#
+# Used by regress and 'make tests'.  Should be run at omi/Unix level.
+
+username=$OMI_USER
+userpasswd=$OMI_PASSWORD
+
+#
+# Clean up previous (if any) running omiserver
+#
+pidFile=./output/var/run/omiserver.pid
```

Comment:
this is going to cause problems as under regress it uses different directory names per configuration it runs